Index: SampleLibrary.Test/UnitTest1.cs
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>namespace SampleLibrary.Test;\r\n\r\npublic class UnitTest1\r\n{\r\n    [Fact]\r\n    public void Test1()\r\n    {\r\n\r\n    }\r\n}
===================================================================
diff --git a/SampleLibrary.Test/UnitTest1.cs b/SampleLibrary.Test/UnitTest1.cs
--- a/SampleLibrary.Test/UnitTest1.cs	
+++ b/SampleLibrary.Test/UnitTest1.cs	
@@ -2,9 +2,14 @@
 
 public class UnitTest1
 {
-    [Fact]
-    public void Test1()
+    [Theory]
+    [InlineData(
+        new string[] { "expected", "another" }, 
+        new string[] { "expected", "another" },
+        new string[] { "expected", "another" }
+        )]
+    public void Test1(string[] optionals, string[] avoids, string[] necessaries)
     {
-
+        
     }
 }
\ No newline at end of file
